src/css/components/help-menu.less

```diff
@@ -0,0 +1,34 @@
+/* =============
+ * Help Menu
+ * =============
+ */
+.help-menu {
+  float: left;
+  margin: 19px;
+  cursor: pointer;
+  color: transparent;
+  background-position: center left;
+  &.dropdown.popover.bottom {
```

Comment:
Yes, our popover-component should be powerful enough that this css adjustments here are not needed.

@Poltergeist Could you use the less-nesting to create the css path?
```css
&.dropdown {
  .popover {
    .bottom {
      ...
    }
  }
}
``` 
With this you can refactor, complement and move around single parts much easier.